### PR TITLE
octave_codes

### PR DESCRIPTION
**Pull request by MichalGQM**
Hey,
I put here the codes:
1. octave_health_check
2. hello_octave
3. configuration
4. set_octave
Also, I added the README file. Let me know if something is not clear in the the README file or if you find any other problem. 

Thanks!

### CODE REVIEW

File: Tutorials/intro-to-octave/README.md [hunk 1]
```diff
@@ -29,13 +49,13 @@ Let's talk about each function separately:
       2. If you want to use an external clock with 100MHz set `external_clock='100MHz'`
       3. If you want to use an external clock with 1GHz set `external_clock='1GHz'`
    2. Sets the up-converters modules.
-      1. LO  - The default LO source is internal. If you want to use an external LO you need to enter `set_octave.py` file and change the LO to the relevant external one (LO1, LO2, LO3, LO4 or LO5).
-      2. Gain - The default is zero gain. You can change it by entering `set_octave.py` file and modify the relevant command.
-      3. Trigger - The default is on. You can change it by entering `set_octave.py` file and modify the relevant command.
-   3. Sets the down-converters modules.
+      1. LO  - The default LO source is internal. If you want to use an external LO you need to enter `set_octave.py` file and change the LO to the relevant external one (LO1, LO2, LO3, LO4 or LO5)
+      2. Gain - The default is zero gain. You can change it by entering `set_octave.py` file and modify the relevant command
+      3. Trigger - The default is on. You can change it by entering `set_octave.py` file and modify the relevant command
```
Comment (TheoLaudatQM) [inline Tutorials/intro-to-octave/README.md:54]: Maybe recall the different option and valid range here as well so that the customers can easily find it

**Review by TheoLaudatQM — COMMENTED** [review]
We are almost there :)

File: Tutorials/intro-to-octave/README.md [hunk 2]
```diff
@@ -2,23 +2,43 @@
...
-1. `health_check_octave.py` runs some basics octave tests.
+1. `octave_introduction.py` contains the basic octave commands.
 2. `hello_octave.py` shows an example of running a program with the octave.
 3. `configuration.py` is the configuration file for `hello_octave.py`.
 4. `set_octave.py` sets all octave parameters.
 
+### `octave_introduction.py`
+In this file you can find the basic octave commands.
+It contains the configuration and its parameters, adds information about the octave and runs octave commands. 
+It is organized in blocks such that:
```
[the remark this answers is not in the document]
Comment (MichalGQM) [reply Tutorials/intro-to-octave/README.md:13]: Thanks!

File: Tutorials/intro-to-octave/README.md [hunk 3]
```diff
@@ -2,23 +2,43 @@
...
+   3. Looking at the octave RF outputs
+   4. Configuring the digital switches
+   5. Configuring the down converter modules
+   6. Calibration
+
+### `hello_octave.py` 
+This file shows an example of how to run a program with a setup contains OPX + octave. It uses the `configuration.py` file and `set_octave.py`  file for all the octave commands.
+
+### `configuration.py` 
+This file contains the config dictionary.
```
[the remark this answers is not in the document]
Comment (MichalGQM) [reply Tutorials/intro-to-octave/README.md:25]: I changed the line to : 
Note that while using your configuration file, there are two things to pay attention to:

What to do you think?

File: Tutorials/intro-to-octave/README.md (hunk 1 above)
Comment (MichalGQM) [reply Tutorials/intro-to-octave/README.md:54]: I added the options for the trigger but the gain might change after Nissim's calibration so I prefer to wait.. Now it's not really the one that is written in the documentation and Yuval is aware of it..

File: Tutorials/intro-to-octave/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (TheoLaudatQM) [reply Tutorials/intro-to-octave/README.md]: That's good

**Review by TheoLaudatQM — APPROVED** [review]
Looks good to me!

**Pull request merged**